### ISSUE TITLE
`ng` should be ignored like `nf` in the netgen setup.

### ISSUE THREAD

**Issue by smunaut** [efabless/IHP-Open-PDK#1]
The `ng` property is "Number of gates" in the IHP models and that's the same as `nf` and so it should also be ignored like `nf` in netgen setup file.

https://ihp-open-pdk-docs.readthedocs.io/en/latest/verification/lvs/04_01_fets.html

**Comment by RTimothyEdwards** [comment]
@mole99 :  Please make this change in the netgen setup file.  This change applies only to MOSFET models, as far as I can tell.  Note that there are a number of parameters passed to the MOSFET subcircuits that are non-standard (i.e., used in subcircuit parameter calculations but not passed directly to the model).  A number of them should _not_ be subcircuit parameters (e.g., `wmin` is _not_ something a user should be able to modify).  Those parameters that show up in netlists (generated by xschem, presumably) should be either handled by netgen or ignored.

The full list of MOSFET parameters that netgen should ignore is, I think:  `ng`, `ad`, `as`, `pd`, `ps`, `rfmode`, and `pre_layout`.   Of course, `l`, `w`, and `m` are handled as usual.

The xschem symbols do not export any parameters other than `l`, `w`, `m`, `ng`, and `rfmode`, but I would put the rest into netgen anyway, especially as the area/perimeter values will be generated for the layout.  I'm not sure why `pre_layout` is even a thing.

**Comment by mole99** [comment]
Hi Tim! I made a few changes in the last update to the netgen file, the parameters that are deleted for mosfets are currently as follows:

    delete ng as ad pd ps trise z1 z2 wmin rfmode pre_layout

Is it correct to ignore `rfmode`? It doesn't seem to be a different device like the cmim/rfcmim, but instead changes the model behavior, just like `pre_layout`.

Sorry @smunaut I missed your issue, I'm now subscribed to this repo so I will be notified in the future.

**Comment by RTimothyEdwards** [comment]
@mole99 :  Yes, `rfmode` is set by xschem when using the "rf" version of the symbol.  There is a specific layout associated with RF operation;  if you look at the devices in `sg13g2_pr.gds` you'll see that the RF devices are strapped across the entire gate and surrounded completely by a guard ring.  I intend to add an "rf" option to the device generators in magic to match the recommended design style.  But the simulation model is the same.  Strapping the gate and adding an enclosing guard ring just guarantee that the model is still correct at high frequencies.

**Comment by mole99** [comment]
Right! Now I see the other layouts in `sg13g2_pr.gds` too ^^
Ok, I'll leave the parameters as they are. I think this issue can be closed then.
